### PR TITLE
Uct gfx

### PR DESCRIPTION
**Pull request by iopq**
This displays the playouts for each coordinate as an analyze command

### CODE REVIEW

**Comment by ujh** [comment]
👍  At least you're doing something useful. Unlike me who is trying to optimize stuff and it doesn't work. ;) Let's wait until https://github.com/rust-lang-nursery/regex/pull/248 is merged and a new version of the regex crate is released before merging it however.

File: src/gtp/mod.rs [hunk 1]
```diff
@@ -78,6 +78,7 @@ impl<'a> GTPInterpreter<'a> {
             "showboard",
             "time_left",
             "time_settings",
+            "uct_gfx",
```
Comment (ujh) [inline src/gtp/mod.rs]: According to the GTP spec these engine specific command should start with a prefix. That's where all these `kgs-*` commands come from. Would it be possible to rename it to `imrscl-uct_gfx`?

File: src/engine/node/mod.rs [hunk 2]
```diff
@@ -378,6 +378,8 @@ impl Node {
         *self.m().color()
     }
 
-
+    pub fn children(&self) -> &Vec<Node> {
+        &(*self).children
```
Comment (ujh) [inline src/engine/node/mod.rs]: Out of curiosity: Is this really necessary? I would have thought that `&self.children` would work just as well.

Comment (iopq) [inline src/engine/node/mod.rs]: you're right, I typed something wrong and it didn't compile so I did this to make the compiler happy, but `&self.children` actually works

File: src/gtp/mod.rs (hunk 1 above)
Comment (iopq) [inline src/gtp/mod.rs]: I just copied Fuego, so yeah, I can just add a prefix

**Comment by ujh** [comment]
Very cool. Thank you!

**Pull request merged**